packages/cli/src/zostso/issue/command/Command.handler.ts

```diff
@@ -36,6 +37,14 @@ export default class Handler extends ZosTsoBaseHandler {
                 addressSpaceOptions: this.mTsoStart
             }
         );
+        const defProc = "IZUPROC";
+        if (params.arguments.logonProcedure && params.arguments.logonProcedure !== defProc && params.arguments.suppressStartupMessages) {
+            this.console.log(
```

Comment:
Should this go to `this.console.error` so it is captured in `stderr` and doesn't impact the output from stdout?